When Beats master is checked out, template generation fails

Workaround is to check out branch 6.5, for example.

But currently, when Beats is checked out on master / 7+, template generation fails with the following error:

```
scripts/template.go:48:41: cannot use "6.0.0" (type string) as type common.Version in argument to template.New
make: *** [template] Error 2
```

--- a/scripts/template.go
+++ b/scripts/template.go
@@ -1,66 +1,66 @@
 package main
 
 import (
-	"fmt"
-	"os"
-	"path/filepath"
+  "fmt"
+  "os"
+  "path/filepath"
 
-	"github.com/elastic/beats/libbeat/common"
-	"github.com/elastic/beats/libbeat/template"
-	"github.com/elastic/go-ucfg/yaml"
+  "github.com/elastic/beats/libbeat/common"
+  "github.com/elastic/beats/libbeat/template"
+  "github.com/elastic/go-ucfg/yaml"
 )
 
 func main() {
 
-	// For the path to be correct, the execution must be from the top directory
-	paths, err := filepath.Glob("./schemas/*.yml")
-	if err != nil {
-		fmt.Printf("Error: %s \n", err)
-		os.Exit(1)
-	}
+  // For the path to be correct, the execution must be from the top directory
+  paths, err := filepath.Glob("./schemas/*.yml")
+  if err != nil {
+    fmt.Printf("Error: %s \n", err)
+    os.Exit(1)
+  }
 
-	fields := common.Fields{}
+  fields := common.Fields{}
 
-	for _, path := range paths {
-		f := common.Fields{}
+  for _, path := range paths {
+    f := common.Fields{}
 
-		cfg, err := yaml.NewConfigWithFile(path)
-		if err != nil {
-			fmt.Printf("Error: %s \n", err)
-			os.Exit(1)
-		}
-		cfg.Unpack(&f)
+    cfg, err := yaml.NewConfigWithFile(path)
+    if err != nil {
+      fmt.Printf("Error: %s \n", err)
+      os.Exit(1)
+    }
+    cfg.Unpack(&f)
 
-		for key, f2 := range f {
-			// The definitions don't have the type group in and the template
-			// generator assumes otherwise keyword as default
-			f[key].Type = "group"
+    for key, f2 := range f {
+      // The definitions don't have the type group in and the template
+      // generator assumes otherwise keyword as default
+      f[key].Type = "group"
 
-			// Moves the docs under base to the top level
-			if f2.Name == "base" {
-				f = f2.Fields
-			}
-		}
+      // Moves the docs under base to the top level
+      if f2.Name == "base" {
+        f = f2.Fields
+      }
+    }
 
-		fields = append(fields, f...)
-	}
+    fields = append(fields, f...)
+  }
 
   // If getting a failure on the following instantiation, check out / update Beats master
   version := common.MustNewVersion("6.0.0")
-	t, err := template.New("1.0.0", "ecs", *version, template.TemplateConfig{})
-	if err != nil {
-		fmt.Printf("Error: %s \n", err)
-		os.Exit(1)
-	}
+  t, err := template.New("1.0.0", "ecs", *version, template.TemplateConfig{})
+  if err != nil {
+    fmt.Printf("Error: %s \n", err)
+    os.Exit(1)
+  }
 
-	// Start processing at the root
-	properties := common.MapStr{}
-	processor := template.Processor{}
-	if err := processor.Process(fields, "", properties); err != nil {
-		fmt.Printf("Error: %s \n", err)
-		os.Exit(1)
-	}
-	output := t.Generate(properties, nil)
+  // Start processing at the root
+  properties := common.MapStr{}
+  processor := template.Processor{}
+  if err := processor.Process(fields, "", properties); err != nil {
+    fmt.Printf("Error: %s \n", err)
+    os.Exit(1)
+  }
+  output := t.Generate(properties, nil)
 
-	fmt.Printf("%s", output.StringToPrint())
+  fmt.Printf("%s", output.StringToPrint())
 }

--- a/scripts/template.go
+++ b/scripts/template.go
@@ -45,7 +45,9 @@ func main() {
 		fields = append(fields, f...)
 	}
 
-	t, err := template.New("1.0.0", "ecs", "6.0.0", template.TemplateConfig{})
+  // If getting a failure on the following instantiation, check out / update Beats master
+  version := common.MustNewVersion("6.0.0")
+	t, err := template.New("1.0.0", "ecs", *version, template.TemplateConfig{})
 	if err != nil {
 		fmt.Printf("Error: %s \n", err)
 		os.Exit(1)

--- a/scripts/template.go
+++ b/scripts/template.go
@@ -1,66 +1,66 @@
 package main
 
 import (
-  "fmt"
-  "os"
-  "path/filepath"
+	"fmt"
+	"os"
+	"path/filepath"
 
-  "github.com/elastic/beats/libbeat/common"
-  "github.com/elastic/beats/libbeat/template"
-  "github.com/elastic/go-ucfg/yaml"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/template"
+	"github.com/elastic/go-ucfg/yaml"
 )
 
 func main() {
 
-  // For the path to be correct, the execution must be from the top directory
-  paths, err := filepath.Glob("./schemas/*.yml")
-  if err != nil {
-    fmt.Printf("Error: %s \n", err)
-    os.Exit(1)
-  }
+	// For the path to be correct, the execution must be from the top directory
+	paths, err := filepath.Glob("./schemas/*.yml")
+	if err != nil {
+		fmt.Printf("Error: %s \n", err)
+		os.Exit(1)
+	}
 
-  fields := common.Fields{}
+	fields := common.Fields{}
 
-  for _, path := range paths {
-    f := common.Fields{}
+	for _, path := range paths {
+		f := common.Fields{}
 
-    cfg, err := yaml.NewConfigWithFile(path)
-    if err != nil {
-      fmt.Printf("Error: %s \n", err)
-      os.Exit(1)
-    }
-    cfg.Unpack(&f)
+		cfg, err := yaml.NewConfigWithFile(path)
+		if err != nil {
+			fmt.Printf("Error: %s \n", err)
+			os.Exit(1)
+		}
+		cfg.Unpack(&f)
 
-    for key, f2 := range f {
-      // The definitions don't have the type group in and the template
-      // generator assumes otherwise keyword as default
-      f[key].Type = "group"
+		for key, f2 := range f {
+			// The definitions don't have the type group in and the template
+			// generator assumes otherwise keyword as default
+			f[key].Type = "group"
 
-      // Moves the docs under base to the top level
-      if f2.Name == "base" {
-        f = f2.Fields
-      }
-    }
+			// Moves the docs under base to the top level
+			if f2.Name == "base" {
+				f = f2.Fields
+			}
+		}
 
-    fields = append(fields, f...)
-  }
+		fields = append(fields, f...)
+	}
 
-  // If getting a failure on the following instantiation, check out / update Beats master
-  version := common.MustNewVersion("6.0.0")
-  t, err := template.New("1.0.0", "ecs", *version, template.TemplateConfig{})
-  if err != nil {
-    fmt.Printf("Error: %s \n", err)
-    os.Exit(1)
-  }
+	// If getting a failure on the following instantiation, check out / update Beats master
+	version := common.MustNewVersion("6.0.0")
+	t, err := template.New("1.0.0", "ecs", *version, template.TemplateConfig{})
+	if err != nil {
+		fmt.Printf("Error: %s \n", err)
+		os.Exit(1)
+	}
 
-  // Start processing at the root
-  properties := common.MapStr{}
-  processor := template.Processor{}
-  if err := processor.Process(fields, "", properties); err != nil {
-    fmt.Printf("Error: %s \n", err)
-    os.Exit(1)
-  }
-  output := t.Generate(properties, nil)
+	// Start processing at the root
+	properties := common.MapStr{}
+	processor := template.Processor{}
+	if err := processor.Process(fields, "", properties); err != nil {
+		fmt.Printf("Error: %s \n", err)
+		os.Exit(1)
+	}
+	output := t.Generate(properties, nil)
 
-  fmt.Printf("%s", output.StringToPrint())
+	fmt.Printf("%s", output.StringToPrint())
 }

